assets/src/blocks/CarouselHeader/CarouselHeaderFrontend.js

```diff
@@ -86,7 +108,7 @@ export const CarouselHeaderFrontend = ({ slides, carousel_autoplay, className })
               active={currentSlide == index}
               ref={element => slidesRef ? slidesRef.current[index] = element : null}
             >
-              <SlideBackground slide={slide} />
+              <SlideBackground slide={pageLoaded ? slide : slides[0]} />
```

Comment:
I added this as a quick way to avoid loading all images. Still need to investigate if it makes sense to do or there are better alternatives.